src/hotspot/cpu/riscv/riscv_v.ad

```diff
@@ -363,6 +373,21 @@ instruct vadd(vReg dst, vReg src1, vReg src2) %{
   ins_pipe(pipe_slow);
 %}
 
+instruct vadd_hfp(vReg dst, vReg src1, vReg src2) %{
+  match(Set dst (AddVHF src1 src2));
+  ins_cost(VEC_COST);
+  format %{ "vadd_hfp $dst, $src1, $src2" %}
+  ins_encode %{
+    assert(UseZvfh, "must");
+    BasicType bt = Matcher::vector_element_basic_type(this);
+    __ vsetvli_helper(bt, Matcher::vector_length(this));
```

Comment:
Question: What is `bt` calculated here? Seems there isn't one for HF16 in `enum BasicType` definition in file src/hotspot/share/utilities/globalDefinitions.hpp. I only see `T_FLOAT` and `T_DOUBLE`, which I don't think is usable here as we need to set SEW=16 for this instruction.